tests/NHSD.BuyingCatalogue.Solutions.Application.UnitTests/NHSD.BuyingCatalogue.Solutions.Application.UnitTests.csproj

```diff
@@ -26,6 +26,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\NHSD.BuyingCatalogue.Infrastructure\NHSD.BuyingCatalogue.Infrastructure.csproj" />
+    <ProjectReference Include="..\..\src\NHSD.BuyingCatalogue.Solutions.API\NHSD.BuyingCatalogue.Solutions.API.csproj" />
```

Comment:
I'm not sure why we need this reference. Could we review this please. The "Application" should know nothing about the "API".